### ISSUE TITLE
Integration does not work on 2023.8

### ISSUE THREAD

**Issue by W-Floyd** [IATkachenko/HA-SleepAsAndroid#53]
### Home Assistant
**Version:** 2023.8 beta
**Installation method** (according to [instructions](https://www.home-assistant.io/installation/)):
[X] Operating System
[ ] Linux VM
[ ] Container
[ ] Supervised
[ ] Core
### Integration

**Installation method:**
[X] HACS
[ ] manual from release
[ ] manual from git
**Integration version:** 1.10.0
**Root topic:** SleepAsAndroid
### Application settings

**topic:** SleepAsAndroid/William
[X] I can see MQTT messages from application at MQTT-server
## How it works now?
 
Integration does not produce any entities
## What do you expect?

Integration should produce entities
## Steps to reproduce
 1. Install integration
 2. Connect SleepAsAndroid
 3. Note lack of functionality 
## Debug log
```
2023-07-30 22:11:39.081 DEBUG (MainThread) [custom_components.sleep_as_android] Got message ReceiveMessage(topic='SleepAsAndroid/William', payload='{"event":"sleep_tracking_started"}', qos=0, retain=0, subscribed_topic='SleepAsAndroid/William', timestamp=datetime.datetime(2023, 7, 31, 3, 11, 39, 81566, tzinfo=datetime.timezone.utc))
2023-07-30 22:11:39.081 DEBUG (MainThread) [custom_components.sleep_as_android] sensor entity_id is SleepAsAndroid_William
2023-07-30 22:11:39.081 DEBUG (MainThread) [custom_components.sleep_as_android.sensor] Processing message ReceiveMessage(topic='SleepAsAndroid/William', payload='{"event":"sleep_tracking_started"}', qos=0, retain=0, subscribed_topic='SleepAsAndroid/William', timestamp=datetime.datetime(2023, 7, 31, 3, 11, 39, 81566, tzinfo=datetime.timezone.utc))
2023-07-30 22:11:39.081 DEBUG (MainThread) [custom_components.sleep_as_android.sensor] New attributes is {'timestamp': 'unavailable', 'label': 'unavailable'}
```

**Comment by nazar-pc** [comment]
2023.8 is now stable, I wish I knew it doesn't work, I would have not upgraded. HA says entity is no longer provided by integration 😕

**Comment by Gibletron** [comment]
Same problem here.
Might have something to do with the MQTT breaking changes:
https://www.home-assistant.io/blog/2023/08/02/release-20238/#breaking-changes

**Comment by Gibletron** [comment]
For those looking for a workaround, you can still create your own MQTT sensor:
```
mqtt:
  sensor:
    - name: "SleepJohn"
      unique_id: sleepjohn_value
      state_topic: "SleepAsAndroid/John"
      icon: mdi:sleep
      force_update: true
      value_template: "{{ value_json.event}}"
```
For my custom templates and automations this works well enough untill there is an official fix

**Comment by theSC** [comment]
Is this being worked on? Should we expect a fix?

**Comment by ndrsfr** [comment]
Same issue for me in 2023.8

**Comment by IATkachenko** [comment]
Will fix it in few days.

Sorry, I have not enough time for the integration support....

**Comment by IATkachenko** [comment]
Done!
Fix release will be available soon.

Right now to fix issue you may manually edit `custom_components/sleep_as_android/sensor.py` to just remove line 195 `"type": None,` from `device_info` property.

**Comment by IATkachenko** [comment]
Fixed in `v2.0.0` (beta now, I hope we will be able to merge #54 today and skip one release / HA restart).

**Comment by Gibletron** [comment]
> Done! Fix release will be available soon.
> 
> Right now to fix issue you may manually edit `custom_components/sleep_as_android/sensor.py` to just remove line 195 `"type": None,` from `device_info` property.

Thank you so much, that's a quick and easy fix!
